src/chip.cs

```diff
@@ -0,0 +1,1767 @@
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeControlSequenceOfOperationWithResponseHandler:")]
+		void ReadAttributeControlSequenceOfOperation (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("writeAttributeControlSequenceOfOperationWithValue:responseHandler:")]
+		void WriteAttributeControlSequenceOfOperation (byte value, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeSystemModeWithResponseHandler:")]
+		void ReadAttributeSystemMode (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("writeAttributeSystemModeWithValue:responseHandler:")]
+		void WriteAttributeSystemMode (byte value, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeClusterRevisionWithResponseHandler:")]
+		void ReadAttributeClusterRevision (ChipResponseHandler responseHandler);
+	}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject), Name="CHIPOnboardingPayloadParser")]
+	[DisableDefaultCtor]
+	interface ChipOnboardingPayloadParser
+	{
+		[Static]
+		[Export ("setupPayloadForOnboardingPayload:ofType:error:")]
+		[return: NullAllowed]
+		ChipSetupPayload SetupPayload (string onboardingPayload, ChipOnboardingPayloadType type, [NullAllowed] out NSError error);
+	}
+
+	interface IChipDevicePairingDelegate {}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[Protocol, Model (AutoGeneratedName = true)]
+	[BaseType (typeof (NSObject), Name="CHIPDevicePairingDelegate")]
+	interface ChipDevicePairingDelegate
+	{
+		[Export ("onStatusUpdate:")]
+		void OnStatusUpdate (ChipPairingStatus status);
+
+		[Export ("onPairingComplete:")]
+		void OnPairingComplete (NSError error);
+
+		[Export ("onPairingDeleted:")]
+		void OnPairingDeleted (NSError error);
+
+		[Export ("onAddressUpdated:")]
+		void OnAddressUpdated (NSError error);
+	}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject), Name="CHIPDeviceController")]
+	[DisableDefaultCtor]
+	interface ChipDeviceController
+	{
+		[Export ("isRunning")]
+		bool IsRunning { get; }
+
+		[Export ("pairDevice:discriminator:setupPINCode:error:")]
+		bool PairDevice (ulong deviceID, ushort discriminator, uint setupPinCode, [NullAllowed] out NSError error);
+
+		[Export ("pairDevice:address:port:discriminator:setupPINCode:error:")]
+		bool PairDevice (ulong deviceID, string address, ushort port, ushort discriminator, uint setupPinCode, [NullAllowed] out NSError error);
+
+		[Export ("pairDeviceWithoutSecurity:address:port:error:")]
+		bool PairDeviceWithoutSecurity (ulong deviceID, string address, ushort port, [NullAllowed] out NSError error);
```

Comment:
Same search and replace. 

```suggestion
		bool PairDeviceWithoutSecurity (ulong deviceId, string address, ushort port, [NullAllowed] out NSError error);
```